app/views/users/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<!--<form accept-charset="UTF-8" action="/users" method="POST">
```

Comment:
on the `action` attribute of the form, you should use the rails URL helpers not a static one